lualib/lua_selectors/transforms.lua

```diff
@@ -441,6 +441,25 @@ Empty string comes the first argument or 'true', non-empty string comes nil]],
     ['description'] = 'Extracts tld from a hostname represented as a string',
     ['args_schema'] = {}
   },
+  -- Converts list of strings to numbers and returns a packed string
+  ['pack_numbers'] = {
+    ['types'] = {
+      ['string_list'] = true
+    },
+    ['map_type'] = 'string',
+    ['process'] = function(inp, t, args)
+      local fmt = args[1] or 'f'
+      local res = {}
+      fun.each(
```

Comment:
One more thing please: could you just use ipairs here instead of `fun.each`. The former has some use cases but it makes the code more difficult here.